.github/CONTRIBUTING.md

```diff
@@ -7,7 +7,7 @@
 * Have a feature request for SDKs? Please post it on [User Voice](https://feedback.azure.com/forums/321918-azure-iot) to help us prioritize.
 * Have a technical question? Ask on [Stack Overflow](https://stackoverflow.com/questions/tagged/azure-iot-hub) with tag “azure-iot-hub”
 * Need Support? Every customer with an active Azure subscription has access to support with guaranteed response time.  Consider submitting a ticket and get assistance from Microsoft support team
-* Found a bug? Please help us fix it by thoroughly documenting it and filing an issue on GitHub (C, Java, .NET, Node.js, Python).
+* Found a bug? Please help us fix it by thoroughly documenting it and filing an issue on GitHub (C#, C, Java, .NET, Node.js, Python).
```

Comment:
Add a note that this repository is for the C# SDK only. Issues found for other languages should be filed under their respective repositories.